src/content/reference/platform-api/annotations/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+linkTitle: Annotationa
+title: Kubernetes resource annotation reference
+description: Overview of Kubernetes resource annotations used by Giant Swarm, and their meaning.
+layout: single
+menu:
+  principal:
+    parent: reference-platform-api
+    identifier: reference-platform-api-annotations
+weight: 500
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+last_review_date: 2025-01-20
+---
+
+**Notice:** Annotations on Kubernetes resources are set by many different parties, and for various reasons. In this overview we explain our reasons for using a relevant set of annotations, and which values or value format is expected. If you are missing information, please consult upstream documentation from Kubernetes etc., or ask a Giant Swarm contact for more information. Also check our corresponding [labels]({{< relref "/reference/platform-api/labels" >}}) reference page.
+
+### network-topology.giantswarm.io/mode
+
+Found on the Cluster (cluster.x-k8s.io/v1beta1) resource for Cluster API provider AWS (CAPA) clusters. Specifies how transit gateways for the cluster will get set up in AWS. Possible values are: `None`, `GiantSwarmManaged`, `UserManaged`.
```

Comment:
The annotation is set on the `AWSCluster` CR.

```suggestion
Found on the AWSCluster resource for Cluster API provider AWS (CAPA) clusters. Specifies how transit gateways for the cluster will get set up in AWS. Possible values are: `None`, `GiantSwarmManaged`, `UserManaged`.
```

Btw, I'd skip mentioning the version of the CR, otherwise we'll have to keep updating this page for no reason (annotations won't be affected by the CR version).

Reply:
> Btw, I'd skip mentioning the version of the CR, otherwise we'll have to keep updating this page for no reason (annotations won't be affected by the CR version).

Works for me. I did it because "Cluster" is such a frequent and unspecific term.

BTW, in workload cluster "operations" on gazelle I see all these annotations in the Cluster resource. Only the ".../mode" one also appears on the AWSCLuster resoure.

Reply:
Our cluster chart only renders these annotations [on the AWSCluster](https://github.com/giantswarm/cluster-aws/blob/main/helm/cluster-aws/templates/_aws_cluster.tpl), so not sure what happened there.

Reply:
OK. I'll accept your changes here and move on.